src/encoded/region_search.py

```diff
@@ -816,6 +816,6 @@ def search_result_actions(request, doc_types, es_results, position=None):
             bucket_has_doc):
         actions['batch_download'] = request.route_url(
             'batch_download',
-            search_params=request.query_string
+            search_params='?' + request.query_string
```

Comment:
I don't think region-search batchdownload works; I think it's disabled in the UI.  I can't remember if it's because it's buggy (throws 500s) or is too slow (timeouts)

Reply:
That's good. Just fixing any mention for consistency.